config/lib/i18n.js

```diff
@@ -41,6 +41,7 @@ i18n
       escapeValue: false, // react already safes from xss
       format
     }
+    // debug: true // show missing translation keys in console.log
```

Comment:
In another separate PR we can add this to config files so that it's easy to turn on/off locally.

Reply:
...actually, take that back; we can't pull backend config files to frontend. :-)

Reply:
If we find a way to do this, it can be helpful. I guess a separate client-only config can do it. API doesn't care about this.